src/actions.js

```diff
@@ -1,6 +1,30 @@
 
 // Websockets actions
 
+function _createActionTypeMap(actionTypes) {
+  return _.reduce(actionTypes, (actionTypeMap, actionType) -> _.set(actionTypeMap, actionType), {});
+}
+
+let noopActionNames = [
+  'WS_CONNECTION_FAILED',
+  'WS_HANDSHAKE_FAILED',
+  'MAKE_TIMERS_STALE',
+  'REQUEST_TIMERS'
+];
+
+export const ActionTypes = _createActionTypeMap(concat(noopActionNames));
+
+function _noopAction(type) {
```

Comment:
I actually think that just writing and exporting this function would suffice. We could then replace the other empty action creator calls with this function.